scripts/common/longhorn.sh

```diff
@@ -53,6 +53,33 @@ function longhorn_install_nfs_utils_if_missing_common() {
     fi
 }
 
+# longhorn_run_pvmigrate calls pvmigrate to migrate longhorn data to a different storage class. if a failure happen
+# then rolls back the original number of volumes and replicas.
+function longhorn_run_pvmigrate() {
+    local longhorn_sc=$1
+    local destStorageClass=$2
```

Comment:
im not a huge fan of mixing cases

Reply:
Yeah, me too. This was a copy and pasta from the Rook to OpenEBS code. Fixed.